### PR TITLE
Adding a new job Opportunity set  views to "1"

### PR DESCRIPTION
**Pull request by Willjobs94**
According to Issue #457, this should avoid that to happen.

Sorry I also changed AccountCountroller.cs but I couldn't let "this" to be there. 😜

### CODE REVIEW

File: EmpleoDotNet/Controllers/JobOpportunityController.cs [hunk 1]
```diff
@@ -52,30 +52,33 @@ public ActionResult Detail(string id)
...
-                _jobOpportunityService.UpdateViewCount(viewModel.Id);
-                CookieHelper.Set(cookieView, viewModel.Id.ToString());
-            }
+            if (IsJobOpportunityPoster(jobOpportunity))
+                return View(nameof(Detail), jobOpportunity);
+
+            var cookieView = $"JobView{jobOpportunity.Id}";
+
+            if (CookieHelper.Exists(cookieView))
+                return View(nameof(Detail), jobOpportunity);
```
Comment (RaulMonteroC) [inline EmpleoDotNet/Controllers/JobOpportunityController.cs]: Puedes unir estos dos ifs en un solo con un ||. Adicionalmente, agregale los brackets para que se vea claro donde comienza y termina el bloque if.

**Comment by Willjobs94** [comment]
@RaulMonteroC done!

File: EmpleoDotNet/EmpleoDotNet.csproj [hunk 2]
```diff
@@ -388,6 +387,8 @@
     <Content Include="Content\css\theme\fonts\glyphicons-halflings-regular.ttf" />
     <Content Include="Content\css\theme\fonts\glyphicons-halflings-regular.woff" />
     <Content Include="Content\css\theme\fonts\glyphicons-halflings-regular.woff2" />
+    <Content Include="Auth.config" />
+    <Content Include="Base_Auth.config" />
```
Comment (eatskolnikov) [inline EmpleoDotNet/EmpleoDotNet.csproj]: Ese archivo no se debe incluir

**Comment by Willjobs94** [comment]
@eatskolnikov  done!

**Pull request merged**